CONTRIBUTING.md

```diff
@@ -60,6 +62,34 @@ The `rustup::currentprocess` module abstracts the global state that is
 `std::env::current_dir` and `std::process::exit` permitting threaded tests of
 the CLI logic; use `process()` rather than those APIs directly.
 
+### Clippy lints
+
+We do not enforce lint status in the checks done by GitHub Actions, because
+clippy is a moving target that can make it hard to merge for little benefit.
+
+We do ask that contributors keep the clippy status clean themselves.
+
+Minimally, run `cargo +nightly clippy --all-targets -- -D warnings` before
+submitting code.
+
+Regular contributors or contributors to particularly OS-specific code should
+also make sure that their clippy checking is done on at least Linux and Windows,
+as OS-conditional code is a common source of unused imports and other small
+lints, which can build up over time.
+
+For developers using BSD/Linux/Mac OS, there are Windows VM's suitable for such
+development tasks for use with virtualbox and other hypervisors are downloadable
+from
+[Microsoft](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/).
+Similarly, there are many Linux and Unix operating systems images available for
+developers whose usual operating system is Windows. Currently Rustup has no Mac
+OS specific code, so there should be no need to worry about Mac VM images.
+
+Clippy is also run in GitHub Actions, in the 'General Checks / Checks` build
```

Comment:
```diff
-Clippy is also run in GitHub Actions, in the 'General Checks / Checks` build
+Clippy is also run in GitHub Actions, in the `General Checks / Checks` build
```

Reply:
This is already merged, I think you can just send a pull request for this.

Reply:
Feel free to do it on my behalf.